playwright/pages/LandingPage.ts

```
@@ -2,6 +2,22 @@ import { expect } from '@playwright/test';
 import type { Locator, Page } from '@playwright/test';
 import { disableCookiePrompt } from '@redhat-cloud-services/playwright-test-auth';
 
+// Timeout constants for e2e tests
+export const TIMEOUTS = {
+  WIDGET_VISIBLE: 60000, // Time to wait for widget to appear in layout
+  MENU_VISIBLE: 15000,   // Time to wait for dropdown menus
```

ESLint

[error] Delete `··`

(prettier/prettier)